status-im/status-react
# [FIX #4327] Migrate to react-native-firebase

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

Fixes #4327

### Summary:

[comment]: # (Summarise the problem and how the pull request solves it)
Migrates Notifications to use invertase/react-native-firebase

### Review notes (optional):


### Testing notes (optional):


### Steps to test:
- Open Status
- Accept Notifications Request
- See if notifications are received



status: ready
